scripts/qa_crowbarsetup.sh

```diff
@@ -1467,6 +1467,10 @@ function onadmin_activate_repositories
 
 function onadmin_bootstrapcrowbar
 {
+    if [[ $upgrademode = "with_upgrade" ]] ; then
+        export cloudsource=$upgrade_cloudsource
+    fi
```

Comment:
I agree. But I call this PR an alternative in case people do not like the other one (https://github.com/SUSE-Cloud/automation/pull/1446)